biosimulators_utils/model_lang/smoldyn/simularium_converter.py

```diff
@@ -0,0 +1,293 @@
+"""Using the Biosimulators side of Smoldyn to generate a modelout.txt Smoldyn file for a specified OMEX/COMBINE archive which then
+    is used to generate a .simularium file for the given simulation. That .simularium file is then stored along with the log.yml and
+    report.{FORMAT} relative to the simulation. Remember: each simulation, while not inherently published, has the potential for publication
+    based purely on the simulation's ability to provide a valid OMEX/COMBINE archive. There exists (or should exist) an additional layer
+    of abstraction to then validate and verify the contents therein.
+"""
+
+
+import os
+from typing import Optional, Tuple, Dict, List, Union
+from abc import ABC, abstractmethod
+import numpy as np
+import pandas as pd
+from simulariumio import (
+    CameraData,
+    UnitData,
+    MetaData,
+    DisplayData,
+    DISPLAY_TYPE,
+    BinaryWriter,
+)
+from simulariumio.smoldyn.smoldyn_data import InputFileData
+from simulariumio.smoldyn import SmoldynConverter, SmoldynData
+from simulariumio.filters import TranslateFilter
+from simulariumio.data_objects.trajectory_data import TrajectoryData, AgentData
+
+
+__all__ = [
+    'CombineArchive',
+    'BiosimulatorsDataConverter',
+    'SmoldynDataConverter',
+]
+
+
+class CombineArchive:
```

Comment:
maybe this should be a "SmoldynCombineArchive" ??

Reply:
This is intended to be a general Combine archive object that will eventually be moved to the global scope.